### PR TITLE
Adding fs scale suite

### PR DESCRIPTION
**Pull request by AmarnatReddy**
Adding fs scale suite

Signed-off-by: Amarnath K <amk@amk.remote.csb>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/cephfs_scale/max_smallfile_snap.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def collect_ceph_details(client, cmd):
 def run(ceph_cluster, **kw):
     """
     Test Cases Covered:
-    CEPH-83573520	Validate the max snapshot that can be created under a root FS sub volume level.
+    CEPH-83574957	Create subvolume and create snapshots until we hit eexception
```
Comment (yogesh-mane) [inline tests/cephfs/cephfs_scale/max_smallfile_snap.py:32]: Need Snapshot under subvolume group here?

File: suites/quincy/cephfs/tier-2_fs_scale.yaml [hunk 2]
```diff
@@ -114,7 +114,7 @@ tests:
       desc: "Snapshots for single volume"
       module: cephfs_scale.max_snapshots_single_volume.py
       name: cephfs Scale test for max Snapshots
-      polarion-id: "CEPH-83573520"
+      polarion-id: "CEPH-83575405"
```
Comment (yogesh-mane) [inline suites/quincy/cephfs/tier-2_fs_scale.yaml:117]: Could we add "cephfs Scale test for max Snapshots under single subvolume" in name? Also similar is needed in description

File: suites/quincy/cephfs/tier-2_fs_scale.yaml [hunk 3]
```diff
@@ -123,10 +123,10 @@ tests:
...
       name: cephfs Scale test for max subvolumes
-      polarion-id: ~
+      polarion-id: "CEPH-83573517"
   - test:
       abort-on-fail: false
       desc: "Max snaps per volume"
       module: cephfs_scale.max_smallfile_snap.py
       name: cephfs Scale test for max Snapshots
-      polarion-id: ~
+      polarion-id: "CEPH-83574957"
```
Comment (yogesh-mane) [inline suites/quincy/cephfs/tier-2_fs_scale.yaml:132]: Could we add snapshots for "subvolume group" in name & description ?

Comment (AmarnatReddy) [reply suites/quincy/cephfs/tier-2_fs_scale.yaml:132]: in script we have not created any subvolume group.
So not mentioned it

File: suites/quincy/cephfs/tier-2_fs_scale.yaml (hunk 2 above)
Comment (AmarnatReddy) [reply suites/quincy/cephfs/tier-2_fs_scale.yaml:117]: Addressed

File: tests/cephfs/cephfs_scale/max_smallfile_snap.py (hunk 1 above)
Comment (AmarnatReddy) [reply tests/cephfs/cephfs_scale/max_smallfile_snap.py:32]: in script we have not created any subvolume group.
So not mentioned it

**Review by yogesh-mane — DISMISSED** [review]

File: tests/cephfs/cephfs_scale/max_smallfile_snap.py (hunk 1 above)
Comment (hkadam134) [reply tests/cephfs/cephfs_scale/max_smallfile_snap.py:32]: subvolume group snapshot feature is yet to come. It's not needed now

**Review by yogesh-mane — APPROVED** [review]

**Pull request merged**